.github/workflows/gradle.yml

```diff
@@ -0,0 +1,20 @@
+name: Gradle Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
WDYT about using a fixed version here, to avoid surprises during the release?

Reply:
Good point, will do.